docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,134 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+**coala** has an awesome testing framework so you can write tests for bears
+with ease.
+
+You can use the following to test your bears:
+
+- ``LocalBearTestHelper.check_validity``
+- ``LocalBearTestHelper.check_results``
+- ``verify_local_bears``
+
+Understanding through examples
+------------------------------
+
+Let us to understand how to write tests for ``MyBear`` written in
+``somelanguage``.
+
+.. code::
+
+    import something_necessary
```

Comment:
how about something totally simple and custom but not vague, like 
```
from coalin.results.Result import Result

class MyBear(LocalBear):

    def run(file, filename):
        if(len(file) > 10):
          yield Result(self, "Too many lines")
``` 
this is just an example off the top of my head , but i believe an example like this would make it easier to understand, and this what i meant by `simpler custom bears`